Tests/kaas/plugin/plugin_static.py

```diff
@@ -0,0 +1,14 @@
+from interface import KubernetesClusterPlugin
+
+
+class PluginStatic(KubernetesClusterPlugin):
+    """
+    Plugin to handle the provisioning of kubernetes
+    using a kubeconfig file
+    """
+
+    def create_cluster(self):
```

Comment:
This class is missing the constructor, and the two methods are missing all arguments.

Reply:
I reworked the `plugin_static.py`  and added the arguments.